src/components/top_bar/top_bar.ts

```diff
@@ -291,4 +292,228 @@ export class TopBar extends Component<Props, SpreadsheetChildEnv> {
   setFontSize(fontSize: number) {
     setStyle(this.env, { fontSize });
   }
+
+  toggleMoreTools() {
```

Comment:
this should be part of the `topBarToolStore`. We can open the "more tools" and another dropdown at the same time ATM